src/main.c

```diff
@@ -330,7 +333,7 @@ int main(int argc, char ** argv){
     PRINT_PAIR("hash", "%X\n", (int) score_hash);
 
     dupprintf("[SCORE%s] Bandwidth %f GB/s : IOPS %f kiops : TOTAL %f\n",
-      opt.is_valid_run ? "" : " INVALID",
+      opt.is_valid_run ? "" : "-invalid",
```

Comment:
with the exception of this change all looks good. I reckon the script outputted INVALID in uppercase as well, that is why it is that way.